src/libs/geometry/src/geom_static.cpp

```diff
@@ -17,6 +19,13 @@ Import library main file
 
 namespace
 {
+
+#ifdef _DEBUG
```

Comment:
maybe 
`#if defined(_DEBUG) || defined(STORM_VALIDATE_COLLISION_DATA)`?
We could have some kind of checked build built in RELEASE but with additional checks like this one. DEBUG builds are so slow that you typically don't want to run them unless you actually want to attach a debugger

Reply:
How about putting it under some kind of `SAFE_MODE` flag? That way we can keep adding additional checks without making the configuration needlessly complex.